cloudevents/generic/options/grpc/protobuf/v1/cloudevent.proto

```diff
@@ -0,0 +1,80 @@
+syntax = "proto3";
+
+package io.cloudevents.v1;
+
+option go_package = "open-cluster-management.io/api/cloudevents/generic/options/grpc/protobuf/v1";
```

Comment:
what is needed if the protobuf message is changed? We need some readme to explain.

Reply:
updated the grpc options to use config similar to mqtt.

Reply:
added a readme file and some comments to guide developer how to update go code after modifying the proto file.